content/applications/hr/recruitment/schedule_interviews.rst

```diff
@@ -0,0 +1,322 @@
+  meeting, and :guilabel:`Only internal users` allows anyone logged into the company database to see
+  the meeting.
+- :guilabel:`Organizer`: the employee who created the meeting is populated in this field. Use the
+  drop-down menu to change the selected employee.
+- :guilabel:`Reminders`: select a reminder from the drop-down menu. Default options include
+  :guilabel:`Notification`, :guilabel:`Email`, and :guilabel:`SMS Text Message`, each with a
+  specific time period before the event (hours, days, etc). The chosen reminder chosen alerts the
+  meeting participants of the meeting, via the selected option at the specified time. Multiple
+  reminders can be selected in this field.
+
+.. image:: schedule_interviews/new-event.png
+   :align: center
+   :alt: A new meeting card with all the details populated, and ready to save.
+
+Send meeting to attendees
+-------------------------
+
+Once changes have been entered on the :ref:`New Event <schedule_interviews/event-card>` pop-up
+window, and the meeting details are correct, the meeting can be sent to the attendees, via email or
+text message, from the expanded event form (what is seen when the :guilabel:`More Options` button is
+clicked on in the :guilabel:`New Event` pop-up window).
+
+To send the meeting via email, click the :icon:`fa-envelope` :guilabel:`Email` button next to the
+:guilabel:`Attendees` field on the expanded meeting form.
+
+A :guilabel:`Contact Attendees` email configurator pop-up window appears. A pre-formatted email,
+using the default :guilabel:`Calendar: Event Update` email template, populates the email body field.
+
+The followers of the job application, as well as the user who created the meeting, are added as
+:guilabel:`Recipients` by default. If needed, add the applicant's email address to the list to send
+the email to the applicant, as well. Make any other desired changes to the email. If an attachment
+is needed, click the :guilabel:`Attachments` button, navigate to the file, then click
+:guilabel:`Open`. Once the email is ready to be sent, click :guilabel:`Send`.
+
+.. image:: schedule_interviews/email-event.png
+   :align: center
+   :alt: Enter the information to send the event via email.
+
+To send the meeting via text message, click the :icon:`fa-mobile` :guilabel:`SMS` button next to
+the :guilabel:`Attendees` field on the expanded meeting form. A :guilabel:`Send SMS Text Message`
+pop-up window appears.
+
+At the top, a blue banner appears if any attendees do not have valid mobile numbers, and lists how
+many records are invalid. If a contact does not have a valid mobile number listed, click
+:guilabel:`Close`, and edit the attendee's record, then redo these steps.
+
+When no warning message appears, type in the message to be sent to the attendees in the
+:guilabel:`Message` field. To add any emojis to the message, click the :icon:`oi-smile-add`
+:guilabel:`(Smileys & Emotions)` icon on the right-side of the pop-up window.
```

Comment:
Update icon label :)
```suggestion
:guilabel:`(smile add)` icon on the right-side of the pop-up window.
```

Reply:
Ahh, ok! When I clicked on the icon, the words that appeared were Smileys & Emotions (just so you didn't think I made up that bizarre term on my own =D)